Some additional changes (#149)
NorskRegnesentral/shapr: vignettes/understanding_shapr.Rmd

The test data given by `x`, whose predicted values we wish to explain, must be provided. Note that 
`x` must be a `data.frame` or a `matrix,` where all elements are finite numerical values.
One must also provide the object returned by `shapr` through the argument `explainer`. 

> The default approach when computing the Shapley values equals the empirical approach (i.e. `approach = "empirical"`). 

Reviewer: ```suggestion
The default approach when computing the Shapley values is the empirical approach (i.e. `approach = "empirical"`). 
```

If you'd like to use a different approach you'll need to set `approach` equal to either `copula` or `gaussian`, or a 

> vector of them. If a vector, a combined approach is used, and element `i` indicates the approach to 

Reviewer: ```suggestion
vector of them, with length equal to the number of features. If a vector, a combined approach is used, and element `i` indicates the approach to 
```

use when conditioning on `i` variables. For more details see [Combined approach](#combined) below.

When computing the kernel SHAP values by `explain`, the maximum number of samples to use in the 
Monte Carlo integration for every conditional expectation is controlled by the argument `n_samples` 
(default equals `1000`). The computation time grows approximately linear with this number. You will also 

> need to pass a numeric value for the argument `prediction_zero`, which represents the prediction value of the 

Reviewer: ```suggestion
need to pass a numeric value for the argument `prediction_zero`, which represents the prediction value when not conditioning on any features.
```

> unseen data. We recommend setting this equal to the mean of the response, but other values, like the mean prediction 

Reviewer: Ref previous line suggestion:
```suggestion
We recommend setting this equal to the mean of the response, but other values, like the mean prediction 
```

of a large test data set is also a possibility. If the empirical method is used, specific settings for that approach, 

> like a fixed $\sigma$ value of vector can be specified through the argument `sigma_vec`. See `?explain` for more 

Reviewer: ```suggestion
like a vector of fixed $\sigma$ values can be specified through the argument `sigma_vec`. See `?explain` for more 
```
